### PR TITLE
Fixes #62 : Added reCaptcha on signup

### LINKED ISSUE #62: Add captcha on the signup page
Captcha helps form submission to be more secure and won't allow automated scripts to fill forms or someone to create robot to fill your form. It is highly recommended to use Captcha. @AkankshaBodhankar 

screen shot 2017-02-06 at 23 17 37

### PR DESCRIPTION
**Pull request by fatimarafiqui**
Fixed #62 - Added Google reCaptcha on registration.php

screen shot 2017-03-09 at 00 21 40

### CODE REVIEW

**Comment by AkankshaBodhankar** [comment]
Can you show the working?

File: includes/settings.php [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?php
```
Comment (AkankshaBodhankar) [inline includes/settings.php]: why have you created a seperate file includes?

Comment (fatimarafiqui) [inline includes/settings.php]: @AkankshaBodhankar  I have created a separate file for google reCaptcha credentials for client and server authentication, it cant be included in the same file as it exposes the risk of key exposure.  Whenever a new user has to set up project for the first time, he needs to generate his /her key and add it to the file. We need to add this to the documentation as well, mentioned in #157 

Posting screenshot in a while.

**Comment by fatimarafiqui** [comment]
@AkankshaBodhankar  Here it is. I have tried my best to explain the flow:

Generating google recaptcha credentials:
1. Client side authentication -



2. Server side authentication -



3. Setting up keys in includes/settings.py



4. Recaptcha on screen -


5. Recaptcha filled correctly leads to account creation.



6. Error alert if captcha not verified-

**Comment by AkankshaBodhankar** [comment]
I was not talking about the flow but the directory includes.Can't you put it up with the other .php files?
Your path shows includes/settings.php

**Comment by fatimarafiqui** [comment]
@AkankshaBodhankar I created the folder includes to keep all the credentials in one directory. I was thinking of keeping Twilio credentials also in the same directory. 
What do you suggest?

**Comment by AkankshaBodhankar** [comment]
@fatimarafiqui Yes that looks good.

**Pull request merged**